scripts/Dockerfile

```diff
@@ -26,13 +22,13 @@ RUN apk add --no-cache \
         cmake \
         g++ \
         make \
-        curl-dev boost-dev libmicrohttpd-dev \
+        linux-headers curl-dev libmicrohttpd-dev \
```

Comment:
Yes. Boost includes at least `<linux/version.h>`. The package is removed after the build.